### DISCUSSION TITLE
Falsche Installation?

### DISCUSSION THREAD

**Discussion by hansfriederbrandt** [RaBa64/OpenDTU#28] [General]
Hallo,

ich habe eine vorinstallierte DTU auf Amazon gekauft. Das Gerät funktioniert und ich erhalte Daten über das Webinterface von meinem Hoymiles hms-800-2t Wechselrichter.

Um in Zukunft die Tageswerte zu sehen, habe des letzten Release heruntergeladen und per OpenDTU Webinterface bei "Firmware Aktualisieren" ausgewählt. Danach wurde das Gerät neu gestartet.

In der "Konfigurationsverwaltung" soll ja nun ein Database-Menu-Eintrag enthalten sein. Diesen sehe ich nicht.

Habe ich bei der Installation einen Fehler gemacht? 
Habe ich etwas anderes nicht bedacht was notwendig ist um diese Software zu nutzen?

**Comment by MartinNK** [comment]
Entweder hat das Update nicht geklappt ... schu mal in der System Seite nch den Daten und vergleiche es mit dem aktuellen Release von openDTU Database.

Es dauert eine ganze Weile bis die ersten Daten in der Grafik und der Database auftauchen ... 1- 2 Stunden.

Hast Du mal angeschaut was deine config.json für eine Konfiguration vorher und nach dem Update enthält?

**Reply by hansfriederbrandt** [reply]
Also das flashen über die OpenDTU Funktion hat nicht funktioniert. Evnetuell muss ich auch die Database Version direkt flashen und die alten Ersetzen?

Naja, ich hab mir jetzt dann doch einen Raspy bestellt um einen MQTT Broker zu bauen und das ganze dann darüber zu managen.

Trotzdem danke!

**Comment by MartinNK** [comment]
Hast Du mal geschaut ob die Partitionen richtig auf dem ESP32 eingerichtet sind.Wie voll ist das Filesystem für die Database bzw. hatte mal auch ein Priblwm mit flashen über das Webinterface da dort nicht gebug Platz war für den Upload der Firmware Datei.MartinVon meinem/meiner Galaxy gesendet
-------- Ursprüngliche Nachricht --------Von: hansfriederbrandt ***@***.***> Datum: 21.07.24  17:37  (GMT+01:00) An: RaBa64/OpenDTU-Database ***@***.***> Cc: MartinNK ***@***.***>, Comment ***@***.***> Betreff: Re: [RaBa64/OpenDTU-Database] Falsche Installation? (Discussion #28) 
Also das flashen über die OpenDTU Funktion hat nicht funktioniert. Evnetuell muss ich auch die Database Version direkt flashen und die alten Ersetzen?
Naja, ich hab mir jetzt dann doch einen Raspy bestellt um einen MQTT Broker zu bauen und das ganze dann darüber zu managen.
Trotzdem danke!

**Comment by hansfriederbrandt** [comment]
Hier 2 Screenshots wie es bei mir aktuell aussieht.
https://imgur.com/a/y83eMDU

**Reply by DerGreifer** [reply]
Hallo, nur zur Info.
Ich habe die DTU eines Bekannten aus der "Ferne" über die OTA-Funktion der DTU auf den Fork- opendtu-database verändert!
Den Fork hatte ich bei mir zuerst auf einem unbenutzten ESP32 geflasht und anschließend im laufenden Betrieb herumgetestet. Glaube immer noch das ich mir einen Schritt hätte sparen können... Egal, Versuch macht klug.
Weitere Info von mir  hier in der Categorie "GENERAL".
PS: Die Config.json musste ich nicht "Nachbearbeiten"!
